pytorch_toolkit/object_detection/model_templates/face-detection/face-detection-0200/compression_config.py

```diff
@@ -0,0 +1,36 @@
+base = dict(
+    find_unused_parameters=True,
+    nncf_config = dict(
+        input_info=dict(sample_size=[1, 3, 256, 256]),
+        compression=[],
+        log_dir='.')
+)
+int8 = dict(
+    optimizer = dict(lr=0.00025),
+    total_epochs = 2,
+    nncf_config = dict(
+        compression=[
+            dict(
+                algorithm='quantization',
+                initializer=dict(
+                    range=dict(num_init_steps=10),
+                    batchnorm_adaptation=dict(num_bn_adaptation_steps=30))),
+            ],
+        )
+)
+sparsity = dict(
+    optimizer = dict(lr=0.0025),
+    total_epochs = 50,
+    nncf_config = dict(
+        compression=[
+            dict(
+                algorithm='magnitude_sparsity',
+                params=dict(
+                    schedule='multistep',
+                    multistep_sparsity_levels=[0.3, 0.5, 0.7],
+                    multistep_steps=[40, 80]))
```

Comment:
This config is controversial. `total_epochs = 50` but `multistep_steps=[40, 80]` which is ahead of training. Another comment is that number of `multistep_sparsity_levels` should correspond to `multistep_steps`.

Reply:
@alexsu52, @vshampor can you also look at this PR.

Reply:
Please, a suggestion here.

I just took one of default configurations from Aleksei Kashapov.

Reply:
Let's use the following schedule:
```
 multistep_sparsity_levels=[0.2, 0.4, 0.5],
 multistep_steps=[10, 20])
```

Reply:
Ok. Done.